django/db/models/fields/__init__.py

```diff
@@ -181,6 +182,30 @@ def _description(self):
 
     description = property(_description)
 
+    def __init_subclass__(cls, **kwargs):
```

Comment:
Emitting at class definition time will make sure the warning is pointed at the proper location and not at query execution time.

Can't use the renamed method deprecation thingy as we need the _shim_ as well.